src/plotman/plotinfo.py

```diff
@@ -0,0 +1,108 @@
+import attr
+
+@attr.s(repr=True, init=False)
```

Comment:
Why no `.__init__()`?  Maybe `order=False` though.  I would have expected that `repr` defaults to effectively being true?